src/main/scala/tile/RocketTile.scala

```diff
@@ -88,11 +88,15 @@ class RocketTile private(
 
   val itimProperty = frontend.icache.itimProperty.toSeq.flatMap(p => Map("sifive,itim" -> p))
 
+  val beuProperty = bus_error_unit.map(d => Map(
+          "sifive,buserror0" -> d.device.asProperty)).getOrElse(Nil)
```

Comment:
I think it would be more idiomatic and readable to call `.toSeq` instead of `.getOrElse(Nil)`.

Also, do we actually want to call this `sifive,buserror0` or should it just be `sifive,buserror`?

Reply:
It should be not be "sifive,buserror0", but simply be "sifive,buserror" as the version as described in the compatible string.

Reply:
ok, i suggested the change, but would like @mhtwn to show what the outputted DTS looks like with this change and get @bsousi5 's final approval before approving the PR

Reply:
@mwachs5 its a plain string - it will show in the dts as is.

Reply:
well i see that string in the DTS above in 2 places. I thought that @bsousi5 was saying in the "compat" he expected it to say sifive,buserror0 but the device type to be sifive,buserror

Reply:
ok, just to not create any confusion, the dts in the release notes (updated above) is how the generated dts looks:
1.  In the cpu component listing: `buserror0` is now -> `buserror`
2. Compatible field in the 'bus_error_unit' component still lists `sifive_buserror0`

Reply:
thanks! @bsousi5 if that looks good to you can you please approve the PR? If that's not enough to make it mergable i will approve it

Reply:
I am good with what @mhtwn commented and shown in the dts.

Reply:
It seems I don't have the permission to hit the approve button. So @mwachs5, you need to approve.

Reply:
I do. So I’ve approved based on your, @richardxia and @mwachs5’s reviews.